datacapturing/src/main/java/de/cyface/datacapturing/ui/CapturingNotification.java

```diff
@@ -80,20 +75,21 @@ public Notification getNotification(DataCapturingBackgroundService context) {
      * Since Android 8 it is necessary to create a new notification channel for a foreground service notification. To save system resources this should only happen if the channel does not exist. This method does just that.
      *
      * @param context The Android <code>Context</code> to use to create the notification channel.
-     * @return The identifier of the created or existing channel.
+     * @param channelId The identifier of the created or existing channel.
      */
     @RequiresApi(api = Build.VERSION_CODES.O)
-    private String createNotificationChannelIfNotExists(final DataCapturingBackgroundService context) {
-        final String channelId = CHANNEL_ID;
+    private void createNotificationChannelIfNotExists(final @NonNull DataCapturingBackgroundService context, final @NonNull String channelId) {
         final CharSequence channelName = "Cyface";
```

Comment:
Wenn wir alle anderen Channel Infos von außen definierbar machen, sollten wir vmtl. auch den ChannelName an gleicher Stelle definieren, sonst steht bei der Stadtradeln App bei den Benachrichtigungseinstellugnen vmtl. noch Notifications: Cyface und keiner weiß, was das ist.

Reply:
Kommt in einem der nächsten Pull Requests. ;)